htdocs/product/card.php

```diff
@@ -2282,6 +2282,10 @@
 					print '<a class="butAction" href="'.$_SERVER["PHP_SELF"].'?action=clone&amp;id='.$object->id.'">'.$langs->trans("ToClone").'</a>';
 				}
 			}
+
+			if ( ! empty($conf->productbatch->enabled) && $object->hasBatch() ) {
```

Comment:
Because it is not an action that has effect on the workflow of the product, it is not consistent to have this link here.
I suggest to replace this with a link on line into the tab Stock, just after "Last movement", having a new line "Last lot / serial" with link to list and a button + to add one new lot batch (your new link)
![image](https://user-images.githubusercontent.com/883887/111703429-e9b0e900-883d-11eb-946f-3d4639ae03be.png)